Backend/Backend.Tests/PlaylistControllerTests.cs

```diff
@@ -417,8 +417,7 @@ public async Task GetUserPlaylists_UserNotFound_ReturnsNotFoundResult()
         var result = await _playlistController.GetUserPlaylists(userId, page, pageSize);
 
         // Assert
-        Assert.IsType<NotFoundObjectResult>(result);
-        Assert.Equal("User does not exist.", (result as NotFoundObjectResult)?.Value);
+        Assert.True(result != null);
```

Comment:
The tests should be updated to reflect the changes in the controller. In this case we would expect a 200 ok response but getting a 500 error response would still make the test pass.